src/base/bittorrent/session.cpp

```diff
@@ -269,6 +288,8 @@ Session::Session(QObject *parent)
     , m_maxUploadsPerTorrent(BITTORRENT_SESSION_KEY("MaxUploadsPerTorrent"), -1, lowerLimited(0, -1))
     , m_isUTPEnabled(BITTORRENT_SESSION_KEY("uTPEnabled"), true)
     , m_isUTPRateLimited(BITTORRENT_SESSION_KEY("uTPRateLimited"), true)
+    , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), m_isUTPEnabled, clampValue(0, 1))
```

Comment:
I **agree** with the bullet points you listed (in sequence), but there is a strong assertion at point 2 which this app doesn't have currently... and I'm not looking for more work, i.e. creating the user manual.

>The use of numbers is much easier from the code POV

It's easier for user to input too, 2 key strokes at most for changing ChokingAlgorithm here.
With string literals, you might get typos, uppercase lowercase issues especially when the user is not fluent in English.

I don't oppose using enums (but there are technical difficulties here) and I just wanted to focus on getting this PR merged...